File: server/db/converter/book.go
package dbconverter

import (
	customdbmodels "github.com/scul0405/my-shop/server/db/customModels"
	dbmodels "github.com/scul0405/my-shop/server/db/models"
	"github.com/scul0405/my-shop/server/internal/dto"
)

func BookModelToDto(book *dbmodels.Book) *dto.BookDTO {
	return &dto.BookDTO{
		ID:         uint64(book.ID),
		CategoryID: uint64(book.CategoryID),
		Name:       book.Name,
		Author:     book.Author,
		Desc:       book.Desc.String,
		Price:      book.Price,
		TotalSold:  book.TotalSold,
		Quantity:   book.Quantity,
		Status:     book.Status,
	}
}

func BookInOrderModelToDto(book *customdbmodels.BookInOrder) *dto.BookInOrderDTO {
	return &dto.BookInOrderDTO{
		ID:            uint64(book.ID),
		CategoryID:    uint64(book.CategoryID),
		Name:          book.Name,
		Author:        book.Author,
		Desc:          book.Desc.String,
		Price:         book.Price,
		TotalSold:     book.TotalSold,
		OrderQuantity: book.OrderQuantity,
		Quantity:      book.Quantity,
		Status:        book.Status,
	}
}

File: client/Entity/Book.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Diagnostics;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entity
{
    public class Book: INotifyPropertyChanged
    {
        public Book() { }

        public int ID { get; set; }
        public string author { get; set;}
        public int category_id { get; set;}
        public string desc { get; set;}
        public string name { get; set;}
        public int price { get; set;}
        public int quantity { get; set;}
        public bool status { get; set;}
        public int total_sold { get; set;}

        public event PropertyChangedEventHandler? PropertyChanged;
    }
}
